style.css

```diff
@@ -343,3 +342,139 @@ p {
 .languages li {
   background: #f7f7f9;
 }
+
+.contact-container {
+  margin: 0 auto;
+  position: relative;
+  max-height: 800px;
+  margin-bottom: 50px;
+  background-color: white;
+}
+
+.form-container {
+  position: relative;
+  background-color: #6070ff;
+  margin-top: 177px;
+  height: 779px;
+  border-top-left-radius: 80px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Great effort working on the contact section of the portfolio project, 👍 👍 Currently i noticed somethings that requires attention in the design

1. There is an extra white space at the bottom of the which makes it look different from the original Figma design. You can get rid of this by increasing the height in the form-container class to `810px`
2. The section is missing a background image as seen in the Figma design, kindly download the the background image to it and be sure it is working perfectly

| Your Design 👇  | Figma Design 👇 |
| ------------- | ------------- |
| ![zaf](https://user-images.githubusercontent.com/78634841/230044327-ed0dd940-5506-4d11-b649-daf58788b0f9.png) | ![temp](https://user-images.githubusercontent.com/78634841/230044449-f2c55dbe-c1cb-4a2c-b330-48e153845d55.png) |

Reply:
Done.